cpapp_backend/cpapp/api/document/views.py

```
@@ -0,0 +1,306 @@
+                # Convert PDF to image if needed
+                if file_type == 'application/pdf':
+                    try:
+                        logger.info("Converting PDF to image")
+                        images = convert_from_path(tmp_path)
+                        if images:
+                            with tempfile.NamedTemporaryFile(delete=False, suffix='.jpg') as img_tmp:
+                                images[0].save(img_tmp.name, 'JPEG')
+                                os.unlink(tmp_path)
+                                tmp_path = img_tmp.name
+                                logger.info(f"PDF converted to image: {tmp_path}")
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)